cmd/secrets-store-csi-driver/main.go

```diff
@@ -159,7 +161,9 @@ func main() {
 	ctx := withShutdownSignal(context.Background())
 
 	// create provider clients
-	providerClients := secretsstore.NewPluginClientBuilder(*providerVolumePath, grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(*maxCallRecvMsgSize)))
+	providerPaths := strings.Split(*additionalProviderPaths, ",")
```

Comment:
Could we also do `strings.TrimSpace(*additionalProviderPaths)` before we split to ensure it works when there are additional spaces?